build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
As per the [ForgeGradle docs](https://github.com/MinecraftForge/ForgeGradle/wiki/Dependencies), the `compile` keyword will build against the jar but only the `embed` keyword does what you are worried about (packaging it in the distribution). That's why the mod doesn't have CodeChickenLib packaged in it and needs it to be downloaded separately.